molecule/default/molecule.yml

```diff
@@ -3,7 +3,7 @@
 dependency:
   name: galaxy
 driver:
-  name: docker
+  name: ${LSR_MOLECULE_DRIVER:-docker}
```

Comment:
The commit message seems to be misleading, it says "use env ..." but it actually adds support for the `LSR_ MOLECULE_DRIVER` variable but does not use it.

Reply:
Does this commit clarify your concern?

Reply:
yes, it addresses this concern. But it seems not to default to docker anymore but to autodetect.

Reply:
> yes, it addresses this concern. But it seems not to default to docker anymore but to autodetect.

If you are running molecule via Travis CI, or running `.travis/runtox` locally, it will autodetect.
If you run `tox -e molecule` you must use `LSR_MOLECULE_DRIVER=podman tox -e molecule` in order to use `podman`.